evaluators/text/HammingDistanceEvaluator/tests/test_hammingdistanceevaluator.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+from .. import HammingDistanceEvaluator
+
+
+@pytest.mark.parametrize('actual, desired, distance',
+    [
+        ('hello', 'hello', 0.0),
```

Comment:
what happens with sentences with space , commas or other punctuations?

Reply:
I think Hamming Distance is supposed to represent the low-level method to compare between two string variables. 
Hence, if we have some input like sentences or other complicated. It is better to use other evaluators to compute the distance.

Reply:
But since we do not have control on how it will be used, it is better to have a test and know how it will behave

Reply:
I got your point. fixed.